TypeCobol.LanguageServer/Completion Factory/CompletionFactory.cs

```diff
@@ -413,9 +413,10 @@ public static IEnumerable<CompletionItem> GetCompletionForQualifiedName(Position
                                 ((firstSignificantToken != null && ((t.StartIndex >= firstSignificantToken.EndColumn && t.Line == firstSignificantToken.Line) || t.Line > firstSignificantToken.Line)) 
                                 || firstSignificantToken == null) 
                                 && ((t.EndColumn <= position.character && t.Line == position.line + 1) || t.Line < position.line + 1))
-                        .Select(t => t.Text);
-                qualifiedName.Reverse();
```

Comment:
Why the call to _qualifiedName.Reverse();_ is no longer needed?

Reply:
It was useless since the beginning because it is the `Reverse()` from LINQ (instead of `Reverse()` from generic List). The Reverse method from LINQ doesn't change the original IEnumerable on which it is called, it produces a new IEnumerable configured to iterate in reverse order. Here the return value of the call is not used so the call itself is useless.